.github/workflows/build.yml

```
@@ -24,6 +24,8 @@ jobs:
           - proton_c
         shield:
           - boardsource3x4
+          - bfo9000_left
+          - bfo9000_right
```

```suggestion
          - bfo9000_left
          - bfo9000_right
          - boardsource3x4
```
I should've spotted this yesterday ... this list should be alphabetical.